pkgs/development/ocaml-modules/graphql/cohttp.nix

```diff
@@ -10,7 +10,7 @@ buildDunePackage rec {
 
   useDune2 = true;
 
-  nativeBuildInputs = [ ocaml-crunch ];
+  nativeBuildInputs = [ crunch ];
```

Comment:
```suggestion
  nativeBuildInputs = [ crunch.bin ];
```

Not tested.

Reply:
I don't see where is the magic but it seems superfluous https://github.com/NixOS/nixpkgs/pull/213222#discussion_r1090083191